app/src/main/java/woowacourse/shopping/data/carts/repository/CartRemoteDataSourceImpl.kt

```
@@ -0,0 +1,221 @@
+                cartId = cartId,
+                authorization = "Basic " + Authorization.basicKey,
+            ).enqueue(
+                object : Callback<Unit> {
+                    override fun onResponse(
+                        call: Call<Unit>,
+                        response: Response<Unit>,
+                    ) {
+                        if (response.isSuccessful) {
+                            onSuccess(response.code())
+                        } else {
+                        }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyElseBlock)